docs/pages/docs/avatar.mdx

```diff
@@ -13,7 +13,7 @@ An image element with a fallback for representing the user.
       Row(
         mainAxisAlignment: MainAxisAlignment.center,
         children: [
-          // With Valid image.
+          // With valid image.
           FAvatar(
             image: const NetworkImage('https://raw.githubusercontent.com/forus-labs/forui/main/samples/assets/avatar.png'),
             fallback: const Text('MN'),
```

Comment:
I think you missed out the `Invalid` below

Reply:
reeeee